src/Kernel32.Shared/Kernel32.cs

```diff
@@ -37,11 +42,6 @@ public static partial class Kernel32
 #pragma warning restore SA1303 // Const field names must begin with upper-case letter
 
         /// <summary>
-        /// Constant for invalid handle value
-        /// </summary>
-        public static readonly IntPtr INVALID_HANDLE_VALUE = new IntPtr(-1);
```

Comment:
Did stylecop want this reordered? I thought all constants were supposed to appear before all static fields.